discord/no-decay/tournaments/october-2019/index.html

```diff
@@ -47,13 +47,13 @@ <h1 class="title is-2">General Rules</h1>
                                         <li>Captains asking for a pause</li>
                                     </ul>
                                 </li>
-                                <li>You are entitled to <b>MAXIMUM</b> five minutes between maps</li>
+                                <li>You are entitled to a <b>MAXIMUM</b> five minutes between maps</li>
                                 <li>
-                                    You are entitled to <b>MAXIMUM</b> five minutes pause time throughout the map. Any time more than this means you will have to resume the match with the members you currently have ready (meaning if you only have 5 players you can sub one of your subs in or play 5v6)
+                                    You are entitled to a <b>MAXIMUM</b> five minutes pause time throughout the map. Any time more than this means you will have to resume the match with the members you currently have ready (meaning if you only have 5 players you can sub one of your subs in or play 5v6)
                                 </li>
                                 <li>If anybody needs to join the lobby, do so when back on the lobby screen rather than in the middle of the match</li>
                                 <li>
-                                    Spectators and substitutes are not allowed to communicate to players mid-map in both text chat and discord. You are able to talk within your pause time or in the lobby
```

Comment:
These were taken from the original doc. I didn't write this. So I'll check with Jelly but I think this should stay.

Reply:
Yep sounds good :) I think it reads a little bit nicer this way but see what Jelly thinks